test/e2e/framework/.import-restrictions

```diff
@@ -0,0 +1,168 @@
+				"k8s.io/kubernetes/pkg/kubelet/container",
+				"k8s.io/kubernetes/pkg/kubelet/dockershim/metrics",
+				"k8s.io/kubernetes/pkg/kubelet/events",
+				"k8s.io/kubernetes/pkg/kubelet/lifecycle",
+				"k8s.io/kubernetes/pkg/kubelet/metrics",
+				"k8s.io/kubernetes/pkg/kubelet/pod",
+				"k8s.io/kubernetes/pkg/kubelet/secret",
+				"k8s.io/kubernetes/pkg/kubelet/sysctl",
+				"k8s.io/kubernetes/pkg/kubelet/types",
+				"k8s.io/kubernetes/pkg/kubelet/util",
+				"k8s.io/kubernetes/pkg/kubelet/util/format",
+				"k8s.io/kubernetes/pkg/kubelet/util/manager",
+				"k8s.io/kubernetes/pkg/kubelet/util/store",
+				"k8s.io/kubernetes/pkg/master/ports",
+				"k8s.io/kubernetes/pkg/registry/core/service/allocator",
+				"k8s.io/kubernetes/pkg/registry/core/service/portallocator",
+				"k8s.io/kubernetes/pkg/scheduler/algorithm",
+				"k8s.io/kubernetes/pkg/scheduler/algorithm/predicates",
+				"k8s.io/kubernetes/pkg/scheduler/algorithm/priorities/util",
+				"k8s.io/kubernetes/pkg/scheduler/api",
+				"k8s.io/kubernetes/pkg/scheduler/metrics",
+				"k8s.io/kubernetes/pkg/scheduler/nodeinfo",
+				"k8s.io/kubernetes/pkg/scheduler/util",
+				"k8s.io/kubernetes/pkg/scheduler/volumebinder",
+				"k8s.io/kubernetes/pkg/security/apparmor",
+				"k8s.io/kubernetes/pkg/security/podsecuritypolicy/seccomp",
+				"k8s.io/kubernetes/pkg/security/podsecuritypolicy/util",
+				"k8s.io/kubernetes/pkg/serviceaccount",
+				"k8s.io/kubernetes/pkg/ssh",
+				"k8s.io/kubernetes/pkg/util/filesystem",
+				"k8s.io/kubernetes/pkg/util/hash",
+				"k8s.io/kubernetes/pkg/util/labels",
+				"k8s.io/kubernetes/pkg/util/metrics",
+				"k8s.io/kubernetes/pkg/util/mount",
+				"k8s.io/kubernetes/pkg/util/node",
+				"k8s.io/kubernetes/pkg/util/parsers",
+				"k8s.io/kubernetes/pkg/util/resizefs",
+				"k8s.io/kubernetes/pkg/util/slice",
+				"k8s.io/kubernetes/pkg/util/system",
+				"k8s.io/kubernetes/pkg/util/taints",
+				"k8s.io/kubernetes/pkg/volume",
+				"k8s.io/kubernetes/pkg/volume/util",
+				"k8s.io/kubernetes/pkg/volume/util/fs",
+				"k8s.io/kubernetes/pkg/volume/util/fsquota",
+				"k8s.io/kubernetes/pkg/volume/util/recyclerclient",
+				"k8s.io/kubernetes/pkg/volume/util/subpath",
+				"k8s.io/kubernetes/pkg/volume/util/types",
+				"k8s.io/kubernetes/pkg/volume/util/volumepathhandler"
+			],
+			"ForbiddenPrefixes": []
+		},
+		{
+			"SelectorRegexp": "k8s[.]io/kubernetes/test/",
+			"AllowedPrefixes": [
+				"k8s.io/kubernetes/test/e2e/framework/auth",
+				"k8s.io/kubernetes/test/e2e/framework/ginkgowrapper",
+				"k8s.io/kubernetes/test/e2e/framework/log",
+				"k8s.io/kubernetes/test/e2e/framework/metrics",
+				"k8s.io/kubernetes/test/e2e/framework/node",
+				"k8s.io/kubernetes/test/e2e/framework/pod",
+				"k8s.io/kubernetes/test/e2e/framework/resource",
+				"k8s.io/kubernetes/test/e2e/framework/ssh",
+				"k8s.io/kubernetes/test/e2e/framework/testfiles",
+				"k8s.io/kubernetes/test/e2e/manifest",
+				"k8s.io/kubernetes/test/e2e/perftype",

```

Comment:
imports that are `k8s.io/kubernetes/test/e2e/` (not `/framework`) are problematic and we need to remove them.